resources/views/dashboard.blade.php

```diff
@@ -1,19 +1,9 @@
 @extends('layouts.app')
 @section('content')
-    <div class="wrapper">
-    <div class="sidebar" data-color="green" >
-    @include('includes.sidebar')
-    </div>
-
-
-
     <div class="main-panel">
             <div class="container-fluid">
               <h4>{{ Auth::user()->firstname}}'s expenses</h4>
-              <div>
-                <table class='table'><thead><tr><th>OWED</th><th>OWES</th><th>TTL</th><th>BAL</th></tr></thead><tbody><tr><td class='text-success'><strong>+0</strong></td><td class='text-danger'><strong>-0</strong></td><td>0</td><td>0</td></tr></tbody>
-                </table></div>
-                <button href="#newExpense" class="btn btn-danger" data-toggle="collapse">New Expense</button>
+              <button href="#newExpense" class="btn btn-danger" data-toggle="collapse">New Expense</button>
               <div class="collapse" id="newExpense">
```

Comment:
We decided on have the summary above the shared expenses table
The expense summary table will never be more than one row so it won't push anything down.